src/site/stages/build/drupal/assets.js

```diff
@@ -1,6 +1,19 @@
 const getDrupalClient = require('./api');
 const cheerio = require('cheerio');
 
+const siteURIs = [
```

Comment:
I could maybe do something fancier to create this list from our drupal config, but this makes what's happening clearer, I think.